chalice/requirements-dev.txt

```diff
@@ -5,4 +5,5 @@ safety==1.8.5
 sshtunnel==0.1.4
 argparse==1.4.0
 behave==1.2.6
-selenium==3.141.0
\ No newline at end of file
+selenium==3.141.0
+coverage
```

Comment:
unpinned requirement. Fine in dev (in that it'll just randomly break for us, not in prod), but it suggests you're manually managing your requirements. Sorry to nitpick!

Reply:
It is a requirement in `requirements-dev.txt` so it is just for us. Also coverage isn't run automatically - yet. I'll add pinning if you think it's a good idea.

Reply:
Pinning Never Hurt Anyone (TM)

Reply:
Pinned!